grafana/k6-docs
# Draft: show-more button fix

This pull request adds condition for "show-more" button in SearchBox:

**Checklist**:

<details>
<summary>Page checked in Chrome:</summary>

**Screenshots**
75 results:
![2022-10-17_19-45-10](https://user-images.githubusercontent.com/75138180/196208608-a4f832f4-519c-4cff-b46b-2354670b8a7e.png)

2 results:
![2022-10-17_19-45-51](https://user-images.githubusercontent.com/75138180/196208728-af557da1-1826-4a8e-bb9e-73946be9ae06.png)

</details>

**Steps to test**
1. Open the [page](https://mdr-ci.staging.k6.io/docs/refs/pull/831/merge)
2. Confirm that everything looks and works as expected
